yarn.lock

```diff
@@ -5882,7 +5887,7 @@ moment-timezone@^0.5.15, moment-timezone@^0.5.34:
   dependencies:
     moment ">= 2.9.0"
 
-"moment@>= 2.9.0", moment@^2.29.1, moment@^2.29.3:
+"moment@>= 2.9.0", moment@^2.29.3:
```

Comment:
Yes, it's still needed, moment is a dependency of `snowflake-sdk@1.6.9` and `mariadb@2.5.6`.

You might wonder what the point of this PR would be if we keep the `moment` dependency.
The advantage will be that it will no longer be an internal dependency of the project.
It will only be needed if the developer needs the `mariadb` peer dependency or if the developer is a sequelize contributor because `snowflake-sdk` is a development dependency.

Reply:
Makes sense :)